app/search/controller.ts

```diff
@@ -41,6 +41,13 @@ export default class SearchController extends Controller {
 
     @tracked page?: number = 1;
 
+    showTooltip1?: boolean;
```

Comment:
These are from PR https://github.com/CenterForOpenScience/ember-osf-web/pull/1877. These lines are necessary for styling the help tutorial but may be reverted when merged together.